src/plugins/home/public/application/components/guided_onboarding/use_case_card.tsx

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { EuiAvatar, EuiCard, EuiText, EuiTitle } from '@elastic/eui';
+
+import { METRIC_TYPE } from '@kbn/analytics';
+
+import { getServices } from '../../kibana_services';
+
+import { i18nTexts, iconTypes, navigateOptions } from './constants';
+
+import './use_case_card.scss';
+
+type UseCase = 'search' | 'observability' | 'security';
+
+export const UseCaseCard = ({ useCase }: { useCase: UseCase }) => {
+  const { application, trackUiMetric } = getServices();
+
+  const onUseCaseSelection = () => {
+    // TODO telemetry for guided onboarding
+    trackUiMetric(METRIC_TYPE.CLICK, `guided_onboarding__use_case__${useCase}`);
+
+    localStorage.setItem(`guidedOnboarding.${useCase}.tourActive`, JSON.stringify(true));
```

Comment:
We're doing this in `localStorage`?  That means a new browser or clearing the cache would defeat the check.  Or is this a "for now" impl?

Reply:
Yes, the decision for 8.4 release is to keep the state in `localStorage`.